core/main/src/broker/endpoint_broker.rs

```diff
@@ -433,14 +462,18 @@ impl EndpointBrokerState {
             return false;
         }
         let rule = found_rule.unwrap();
-        let broker = broker_sender.unwrap();
-        let updated_request = self.update_request(&rpc_request, rule, extn_message);
-        tokio::spawn(async move {
-            if let Err(e) = broker.send(updated_request.clone()).await {
-                // send some rpc error
-                callback.send_error(updated_request, e).await
-            }
-        });
+
+        if rule.alias == "static" {
+            self.handle_static_request(rpc_request, extn_message, rule, callback);
```

Comment:
I think we want to handle static rules before we check if `broker_sender.is_none()`, because static rules will not have a broker right?

Reply:
Rearranged the conditions.